From: Aaron Seigo <aseigo@kde.org>
Date: Tue, 05 Mar 2013 17:03:49 +0000
Subject: containments have configuration by default
X-Git-Tag: v4.10.2
X-Git-Url: http://quickgit.kde.org/?p=kdelibs.git&a=commitdiff&h=8d4f958586ab29bcfa9c70bb919540839c9bd2ac
---
containments have configuration by default

prevents this from getting greyed out permenantly on failure
---


--- a/plasma/containment.cpp
+++ b/plasma/containment.cpp
@@ -118,7 +118,6 @@
     setPos(0, 0);
     setBackgroundHints(NoBackground);
     setContainmentType(CustomContainment);
-    setHasConfigurationInterface(false);
 }
 
 Containment::Containment(QObject *parent, const QVariantList &args)
@@ -129,7 +128,6 @@
     //          that requires a scene, which is not available at this point
     setPos(0, 0);
     setBackgroundHints(NoBackground);
-    setHasConfigurationInterface(false);
 }
 
 Containment::Containment(const QString &packagePath, uint appletId, const QVariantList &args)
@@ -140,7 +138,6 @@
     //          that requires a scene, which is not available at this point
     setPos(0, 0);
     setBackgroundHints(NoBackground);
-    setHasConfigurationInterface(false);
 }
 
 Containment::~Containment()

